ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -0,0 +1,26 @@
+import ivy
+
+
+def floor(
+    x,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="k",
+    dtype=None,
+    subok=True,
+    signature=None,
+    extobj=None,
+):
+    if dtype:
+        x = ivy.astype(ivy.array(x), ivy.as_ivy_dtype(dtype))
+    else:
+        dtype = x.dtype
+    if out is None:
```

Comment:
this check is not required as `out` can also be accepted as `None`

Reply:
Thx for review. 
Yes, but this check's purpose to replicate numpy floors behavior by assigning NaN values to fields where "where" variable is False if out is None. See [docs](https://numpy.org/doc/stable/reference/generated/numpy.floor.html)

Reply:
oh okay thanks for pointing out!